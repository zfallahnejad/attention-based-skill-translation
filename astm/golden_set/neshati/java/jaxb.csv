383861,jaxb
303810,jaxb
21234,jaxb
1143825,jaxb
2107876,jaxb
2587435,jaxb
3364187,jaxb
552759,jaxb
438154,jaxb
886533,jaxb
103154,jaxb
