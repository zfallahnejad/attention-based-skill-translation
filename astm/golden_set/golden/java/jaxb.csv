383861,951,640,0.5977372721558768
303810,81,113,0.4175257731958763
21234,78,58,0.5735294117647058
1143825,22,22,0.5
2107876,21,27,0.4375
2587435,16,4,0.8
3364187,15,21,0.4166666666666667
552759,15,17,0.46875
103154,13,2,0.8666666666666667
886533,13,12,0.52
438154,13,6,0.6842105263157895
116509,10,8,0.5555555555555556
